### PR TITLE
Component Issues Details visual changes

### PR DESCRIPTION
**Pull request by barbelity**

### CODE REVIEW

File: src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssueDetails.java [hunk 1]
```diff
@@ -17,8 +17,6 @@ private ComponentIssueDetails(DependenciesTree node) {
         super(node);
         Issue topIssue = node.getTopIssue();
         addText("Top Issue Severity:", StringUtils.capitalize(topIssue.getSeverity().toString()));
-        addText("Top Issue Type:", StringUtils.capitalize(topIssue.getIssueType()));
-        addText("Issues Count:", String.valueOf(node.getIssueCount()));
```
Comment (yahavi) [inline src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssueDetails.java:21]: Please don't delete the issues count.

File: src/main/java/com/jfrog/ide/idea/ui/issues/IssuesTab.java [hunk 2]
```diff
@@ -147,7 +147,9 @@ public void updateIssuesTable() {
                 .map(scanManager -> scanManager.getFilteredScanIssues(FilterManagerService.getInstance(mainProject), selectedNodes))
                 .flatMap(Collection::stream)
                 .collect(Collectors.toSet());
-        issuesTable.updateIssuesTable(issueSet);
+
+        Set<String> selectedNodeNames = selectedNodes.stream().map(node -> node.toString()).collect(Collectors.toSet());
```
Comment (yahavi) [inline src/main/java/com/jfrog/ide/idea/ui/issues/IssuesTab.java:151]: Just to get rid of the IDE warning:
```suggestion
        Set<String> selectedNodeNames = selectedNodes.stream().map(DefaultMutableTreeNode::toString).collect(Collectors.toSet());
```

File: src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssuesTable.java [hunk 3]
```diff
@@ -46,11 +46,13 @@ void updateIssuesTable(Set<Issue> issueSet) {
 
     private void resizeTableColumns() {
         int tableWidth = getParent().getWidth();
-        tableWidth -= getColumnModel().getColumn(SEVERITY.ordinal()).getPreferredWidth();
-        tableWidth -= getColumnModel().getColumn(ISSUE_TYPE.ordinal()).getPreferredWidth();
+
+        TableColumn severityCol = getColumnModel().getColumn(SEVERITY.ordinal());
+        severityCol.setPreferredWidth(severityCol.getPreferredWidth() / 2);
```
[the remark this answers is not in the document]
Comment (barbelity) [reply src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssuesTable.java:51]: Moved to center.

File: src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssuesTable.java [hunk 4]
```diff
@@ -46,11 +46,13 @@ void updateIssuesTable(Set<Issue> issueSet) {
 
     private void resizeTableColumns() {
```
[the remark this answers is not in the document]
Comment (barbelity) [reply src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssuesTable.java:47]: Our previous specification was 40-60 between components and summary. Do you think 30-70 is good?

File: src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssueDetails.java (hunk 1 above)
Comment (barbelity) [reply src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssueDetails.java:21]: @eyalbe4 we talked about removing the 'Issues Count' property, what do you think?

File: src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssuesTable.java (hunk 4 above)
[the remark this answers is not in the document]
Comment (yahavi) [reply src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssuesTable.java:47]: Yes. Better.

File: src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssuesTable.java [hunk 5]
```diff
@@ -30,8 +30,8 @@
         setAutoResizeMode(AUTO_RESIZE_OFF);
     }
 
-    void updateIssuesTable(Set<Issue> issueSet) {
-        TableModel model = new IssuesTableModel(issueSet);
+    void updateIssuesTable(Set<Issue> issueSet, Set<String> selectedComponents) {
+        TableModel model = new IssuesTableModel(issueSet, selectedComponents);
         TableRowSorter<TableModel> sorter = new TableRowSorter<>(model);
         sorter.setComparator(SEVERITY.ordinal(), Comparator.comparing(o -> ((Severity) o)));
```
[the remark this answers is not in the document]
Comment (barbelity) [reply src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssuesTable.java:36]: Good input, added a comparator for the 'Component' column.
Take a look at ComponentIssuesTable.java.

File: src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssuesTable.java [hunk 6]
```diff
@@ -30,30 +32,51 @@
...
+                return s1.compareTo(s2);
+            }
+            if (selectedComponents.contains(s1)) {
+                return -1;
+            }
+            if (selectedComponents.contains(s2)) {
+                return 1;
+            }
+            return s1.compareTo(s2);
+        };
```
Comment (yahavi) [inline src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssuesTable.java:60]:
```suggestion
        sorter.setComparator(COMPONENT.ordinal(), Comparator.comparing(s -> selectedComponents.contains(s.toString()) ? -1 : 0));
```

Comment (barbelity) [reply src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssuesTable.java:60]: 👍  + added second comparator by component name.

File: src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssueDetails.java (hunk 1 above)
Comment (yahavi) [reply src/main/java/com/jfrog/ide/idea/ui/issues/ComponentIssueDetails.java:21]: OK I talked with him. We agreed to remove the issues count.

**Pull request merged**